src/driver/hermes_mod.c

```diff
@@ -55,12 +55,102 @@ static const struct pci_device_id pci_ids[] = {
 };
 MODULE_DEVICE_TABLE(pci, pci_ids);
 
+static void irq_teardown(struct xdma_dev *xdev)
+{
+	irq_msix_channel_teardown(xdev);
+}
+
+static int irq_setup(struct xdma_dev *xdev)
+{
+	return xdma_irq_setup(xdev);
+}
```

Comment:
Same here... What's the point of another set of wrappers to rename the functions?